cocos/base/CCUserDefault.h

```diff
@@ -202,6 +202,14 @@ class CC_DLL UserDefault
      */
     virtual void flush();
 
+    /**
+    * delete any value by key, if the key doesn't exist, will return true.
```

Comment:
What does it mean "if the key doesn't exist, will return true"?
If the parameter is nullptr, you return false. If the key doesn't exist, you return false.
I think we can't tell these two cases apart, right?